### PR TITLE
Buggyamb - ASP.NET Core debugging scenarios

### PR DESCRIPTION
**Pull request by ahmetmithat**
## Summary

BuggyAmb is just a very buggy ASP.NET Core Razor Pages based application. This application was intentionally created as a buggy application so it can be used as a learning resource to practice troubleshooting some sample problem scenarios:

* High CPU or low CPU performance problems, deadlocks.
* High memory issues.
* Crash issues and unexpected exceptions.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4740) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4740) before we can accept your contribution.<br/><br/>:x: ahmetmithat [sign now](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4740)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/samples?pullRequest=4740) it.</sub>

**Review by CamSoper — CHANGES_REQUESTED** [review]
Overall, looks good. I'll provide suggestions.

**Review by CamSoper — CHANGES_REQUESTED** [review]
Overall, looks great. Please remove the `.gitignore` and `.gitattributes` files, as those are handled at the repo level.

Since you're publishing an article elsewhere to cover the main content, I'm not too worried about doing an edit pass on the README. I think it's fine for a README, but I'd like @IEvangelist to give his opinion, too.

Awesome work, Ahmet!

File: core/tutorials/buggyamb/Docs/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Installation instructions
```
Comment (IEvangelist) [inline core/tutorials/buggyamb/Docs/README.md:1]: In order for this sample to be discoverable we need to have the appropriate front matter. See https://review.docs.microsoft.com/en-us/help/contribute/samples/concepts/readme-template?branch=master

Also, I *think* this README should be at the root of the sample, so you'd have to move it down a directory - and update the refs to the other MD files. Here is a simple README with the minimal required front matter and content:

- https://github.com/dotnet/samples/blob/main/core/logging/custom-formatter/README.md
- https://docs.microsoft.com/en-us/samples/dotnet/samples/csharp-logging-fundamentals

**Review by IEvangelist — COMMENTED** [review]
Need a bit of work done to the _README.md_, other than that - let's shipit.

**Comment by ahmetmithat** [comment]
Hi Cam,

many thanks, I just removed the .gitignore and .gitattributes files.

Thanks,
AMB

**Comment by ahmetmithat** [comment]
Sorry, please ignore previous change, I removed wrong files but fixed it.

I have removed the correct .gitignore and .gitattributes files and pushed to the origin.

Thanks,
AMB

From: Ahmet Mithat Bostanci
Sent: Thursday, August 19, 2021 4:21 PM
To: dotnet/samples ***@***.***>; dotnet/samples ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [dotnet/samples] Buggyamb - ASP.NET Core debugging scenarios (#4740)

Hi Cam,

many thanks, I just removed the .gitignore and .gitattributes files.

Thanks,
AMB

**Review by CamSoper — APPROVED** [review]

**Pull request merged**